Projects/b_u585i_iot02a_tfm/tfm.mk

```diff
@@ -56,7 +56,9 @@ NS_REGION_SIGNING_KEY = ${TFM_SRC_PATH}/bl2/ext/mcuboot/root-RSA-3072_1.pem
 # Version number definition
 ###############################################################################
 SPE_VERSION = "1.5.0"
+SPE_VERSION_OTA = "2.5.0"
```

Comment:
Why have separate SPE_VERSION and SPE_VERSION_OTA definitions?

Reply:
We need a higher version for OTA image than the one  loaded to flash otherwise version validation for rollback protection will fail.

Reply:
That's not the point of generating a separate ota image. All of the build artifacts from a given build need to have the same version number.

Reply:
Ok I agree. The only point I added here was it will be easier for testing during OTA development phase. Changed to single version for both.